src/classes/CW_CampaignSitesTriggerHandler.cls

```diff
@@ -0,0 +1,109 @@
+global class CW_CampaignSitesTriggerHandler {
+/**
+* File Name   :    CW_CampaignSitesTriggerHandler.cls
+* Description :    
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 21/05/2014 Chris Yorskton Initial
+* 0.2 28/05/2014 Klea Kolaric	Fixing trigger chunking
+* 0.3 16/06/2014 Chris Yorskton Added the aggregation of Brands on Campaign object
+*
+*/	
+	global static Boolean futureCalled = false;	// avoid multiple calls to future method
+	
+	private Boolean m_isExecuting = false;
+	
+    global CW_CampaignSitesTriggerHandler(Boolean isExecuting){
+        m_isExecuting = isExecuting;
+    }
+    
+    global boolean IsTriggerContext{
+        get{ return m_isExecuting; }
+    }
+	
+	global void OnAfterInsert( CW_Campaign_Sites__c[] newObjects, Map<Id,CW_Campaign_Sites__c> newObjectMap ) {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.